packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputElement.tsx

```diff
@@ -176,6 +181,10 @@ export const TransactionReviewOutputElement = forwardRef<
         const isActive = state === 'active';
 
         const showMultiIndicator = lines.length > 1;
+        const displaySymbolControlled =
```

Comment:
`displaySymbol` here is `displaySymbol?: NetworkSymbolExtended;` that does not sound right